posix.mak

```diff
@@ -127,7 +127,27 @@ SRCS:=$(subst \,/,$(SRCS))
 # NOTE: a pre-compiled minit.obj has been provided in dmd for Win32	 and
 #       minit.asm is not used by dmd for Linux
 
-OBJS= $(ROOT)/errno_c.o $(ROOT)/threadasm.o
+OBJS= $(ROOT)/threadasm.o
+
+# checks if linking against stdc not disabled
+ifndef DRUNTIME_NOSTDC
+include mak/STDC
```

Comment:
Since it affects all files in two packages, wouldn't it be better to use [filter-out](https://www.gnu.org/software/make/manual/html_node/Text-Functions.html) ?

Reply:
I don't think that this is very important. My option will help to easily move these packages to another location at future .

Reply:
And I saw that filter-out is already used. I'll redo it now.